client/src/components/useFeatureFlag.ts

```diff
@@ -41,6 +41,10 @@ const BATCH_INVENTORY_CONFIGS: {
   'a67791e3-90a0-4d4e-a5e7-929f82bf4ce6': {
     showBallotManifest: false,
   },
+  // Maryland (Brian local development)
+  'b6c82242-7737-49dd-b315-cc34f8ad8de0': {
+    showBallotManifest: true,
```

Comment:
After checking with Ginny, we should set this field to `false`. For context, an org's existence in this object is what enables the batch audit file prep tool overall. And the true/false status of this sub-field enables the sub-feature of being able to download a generated ballot manifest. If this is false, you can only download a generated candidate-totals-by-batch file. I confirmed with Ginny that we only want to auto-generate a candidate-totals-by-batch file for MD. We still want them creating their ballot manifest manually.